### ISSUE TITLE
Close the window and it throws error: FATAL EXCEPTION: main and killed my app completely.

### ISSUE THREAD

**Issue by asongkai** [pvsvamsi/SystemAlertWindow#39]
When try to press reject call button and call:

```
Future<void> _checkPermissions() async {
    await SystemAlertWindow.checkPermissions;
    ReceivePort _port = ReceivePort();
    IsolateManager.registerPortWithName(_port.sendPort);
    _port.listen((dynamic callBackData) {
      String tag= callBackData[0];
      switch (tag) {
        case "reject_call":
          print("Call reject");
          SystemAlertWindow.closeSystemWindow();
          break;
        case "accept_call":
          print("Accept Call");

          break;
      }
    });
    SystemAlertWindow.registerOnClickListener(callBackFunction);
  }
```

Throw error and killed my app

```
V/SystemAlertWindowPlugin(32685): invoking callback for tag reject_call
V/SystemAlertWindowPlugin(32685): Invoking on method channel
I/flutter (32685): Got tag reject_call
I/flutter (32685): Call reject
I/SystemAlertWindowPlugin(32685): Invoke call back success
D/AndroidRuntime(32685): Shutting down VM
E/AndroidRuntime(32685): FATAL EXCEPTION: main
E/AndroidRuntime(32685): Process: com.oudomsup.ocwa, PID: 32685
E/AndroidRuntime(32685): java.lang.AssertionError
E/AndroidRuntime(32685): 	at in.jvapps.system_alert_window.services.WindowServiceNew.onStartCommand(WindowServiceNew.java:95)
E/AndroidRuntime(32685): 	at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:4204)
E/AndroidRuntime(32685): 	at android.app.ActivityThread.access$2100(ActivityThread.java:231)
E/AndroidRuntime(32685): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1984)
E/AndroidRuntime(32685): 	at android.os.Handler.dispatchMessage(Handler.java:107)
E/AndroidRuntime(32685): 	at android.os.Looper.loop(Looper.java:214)
E/AndroidRuntime(32685): 	at android.app.ActivityThread.main(ActivityThread.java:7682)
E/AndroidRuntime(32685): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(32685): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:516)
E/AndroidRuntime(32685): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
I/Process (32685): Sending signal. PID: 32685 SIG: 9
```

**Comment by asongkai** [comment]
Hi any comment on this?

**Comment by asongkai** [comment]
Any update on my issue?

**Comment by asongkai** [comment]
Hi,

Any help on this?

**Comment by pvsvamsi** [comment]
Fixed in 0.4.1+1
